src/index.js

```diff
@@ -66,7 +66,8 @@ const filesProto = {
   statByPath: files.statByPath,
   downloadById: files.downloadById,
   downloadByPath: files.downloadByPath,
-  getDownloadLink: files.getDownloadLink,
```

Comment:
What do you think about keeping `getDownloadLink` pointing to `getDownloadLinkByPath`, to avoid breaking a lot of things in Files and Photos ? We could comment it as `deprecated` and remove it in a couple of release.